pyproject.toml

```diff
@@ -1,18 +1,44 @@
-[tool.poetry]
+[project]
 name = "tortoise-orm"
 version = "0.24.1"
 description = "Easy async ORM for python, built with relations in mind"
-authors = ["Andrey Bondar <andrey@bondar.ru>", "Nickolas Grigoriadis <nagrigoriadis@gmail.com>", "long2ice <long2ice@gmail.com>"]
-license = "Apache-2.0"
+authors = [{name="Andrey Bondar", email="andrey@bondar.ru"}, {name="Nickolas Grigoriadis", email="nagrigoriadis@gmail.com"}, {name="long2ice", email="long2ice@gmail.com"}]
+license = {text="Apache-2.0"}
 readme = "README.rst"
+keywords = ["sql", "mysql", "postgres", "psql", "sqlite", "aiosqlite", "asyncpg", "relational", "database", "rdbms", "orm", "object mapper", "async", "asyncio", "aio", "psycopg"]
+include = ["CHANGELOG.rst", "LICENSE", "README.rst"]
+# classifieres is dynamic because poetry will create Python classifiers automatically with value in `tool.poetry` section
+dynamic = [ "classifiers" ]
+requires-python = ">=3.9"
+dependencies = [
+    "pypika-tortoise (>=0.5.0,<1.0.0); python_version < '4.0'",
+    "iso8601 (>=2.1.0,<3.0.0); python_version < '4.0'",
```

Comment:
Why do you have to specify the Python version?

Reply:
If this https://github.com/tortoise/pypika-tortoise/pull/26 merged, pypika-tortoise will be no need to specify the Python version.